content/10.diagnostics.md

```diff
@@ -114,8 +114,8 @@ In dPCR, a sample is partitioned into thousands of compartments, such as nanodro
 This design allows for a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for, allowing for absolute target quantification through Poisson statistics.
 While dPCR equipment is not yet as common as that for qPCR, dPCR for DNA targets generally achieves higher sensitivity than other PCR technologies while maintaining high specificity, though sensitivity is slightly lower for RNA targets [@doi:10.3390/s18041271].
 High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
-Suo et al. (2020) [@doi:10.1080/22221751.2020.1772678] performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection on 57 samples, comprised by 43 samples from suspected positive patients and 14 from supposed convalescents, that had all tested negative for SARS-CoV-2 using qPCR.
-Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive.
+Suo et al. [@doi:10.1080/22221751.2020.1772678] performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection on 57 samples, 43 samples from suspected positive patients and 14 from supposed convalescents, that had all tested negative for SARS-CoV-2 using qPCR.
+Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive. <!-- Where did the 35 come from? -->
```

Comment:
`33 out of 35` is wrong in the current context, the numbers come from the first version of the preprint (https://www.medrxiv.org/content/10.1101/2020.02.29.20029439v1). I think the numbers in the entire section need to be double-checked.